### PR TITLE
Fixed json schemata

### PR DESCRIPTION
**Pull request by che85**
validation based on http://www.jsonschemavalidator.net/

### CODE REVIEW

**Comment by che85** [comment]
@fedorov  will need to change the reference links

File: doc/common-schema.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "@context": "http://qiicr.org/dcmqi/contexts/dcmqi.jsonld",
-  "id" : "http://qiicr.org/schema/dcmqi/common-schema.json",
+  "id" : "https://raw.githubusercontent.com/che85/dcmqi/fixSchema/doc/common-schema.json#",
```
Comment (fedorov) [inline doc/common-schema.json]: point to a location under qiicr/dcmqi instead?

Comment (che85) [inline doc/common-schema.json]: That's what I just commented on my last comment for the PR ;)

File: doc/seg-schema.json [hunk 2]
```diff
@@ -1,152 +1,197 @@
...
-        "SegmentDescription": { "type" : "string", "required" : false},
-
-        "SegmentedPropertyCategoryCodeSequence": {
-          "allOf" : [
-            {"$ref" : "http://qiicr.org/schema/dcmqi/common-schema.json#/definitions/codeSequence"},
-            {"properties" :
-             {"CodeValue" : {"default": "T-D0050"},
-              "CodingSchemeDesignator" : {"default": "SRT"},
-              "CodeMeaning" : {"default": "Tissue"}}
+  "properties": {
```
Comment (fedorov) [inline doc/seg-schema.json]: why is this "properties" here?

File: doc/seg-schema.json [hunk 3]
```diff
@@ -1,152 +1,197 @@
...
+          ],
+          "default": "Reader1"
+        },
+        "ClinicalTrialSeriesID": {
+          "allOf": [
+            {
+              "$ref": "https://raw.githubusercontent.com/che85/dcmqi/fixSchema/doc/common-schema.json#/definitions/LO"
             }
           ],
-          "required" : true
```
Comment (fedorov) [inline doc/seg-schema.json]: why did you remove required? is it redundant if there is a default?

File: doc/seg-schema.json [hunk 4]
```diff
@@ -1,152 +1,197 @@
...
+              "26565",
+              "37722"
+            ]
+          }
         }
       }
     }
   }
-}
+}
```
Comment (fedorov) [inline doc/seg-schema.json]: github doesn't like that you don't have newline in the end ;-)

File: doc/seg-schema.json (hunk 2 above)
Comment (che85) [inline doc/seg-schema.json]: Without it, the schema is not valid

File: doc/seg-schema.json [hunk 5]
```diff
@@ -1,152 +1,197 @@
...
+    "segmentAttributes": {
+      "type": "array",
+      "items": {
+        "type": "object",
+        "required": [
+          "SegmentedPropertyCategoryCodeSequence",
+          "SegmentedPropertyTypeCodeSequence",
+          "SegmentAlgorithmType"
+        ],
+        "properties": {
```
Comment (che85) [inline doc/seg-schema.json]: @fedorov all required attributes moved into one array on top of properties, not for each separate

Comment (fedorov) [inline doc/seg-schema.json]: oh, ok

LabelID should also be required

Comment (che85) [inline doc/seg-schema.json]: done

File: doc/seg-schema.json (hunk 2 above)
Comment (fedorov) [inline doc/seg-schema.json]: oh, I see, I missed it - I thought it is present only in this schema, but it also shows up for pm-schema. Alright

**Comment by fedorov** [comment]
Looks good! I am merging, since I don't expect anything new on CI for json changes
